README.md

```diff
@@ -448,6 +448,40 @@ DispatchRider.config do |config|
   config.error_handler = DispatchRider::AirbrakeErrorHandler
 end
 ```
+## Scheduling Job
+
+Scheduled jobs currently require `ActiveRecord`. Support for other ORM will be added in the future depending on demand.
+
+1. Add and run the DB migration below:
+  ```ruby
+  class CreateScheduledJobsTable < ActiveRecord::Migration
+    include DispatchRider::ScheduledJob::Migration
+
+    def change
+      create_scheduled_jobs_table
+    end
+  end
+  ```
+
+2. Run publishing.
+  ```ruby
+  # Run once
+
+  # Ideally run on a cron. Where the cron is responsible for the publishing
+  # frequency. Any jobs due at the time this is run by the cron  will be
+  # published.
+  DispatchRider::ScheduledJob.publish_due_jobs
+
+  # Loop. Warning: Loops are blocking. Run this on a separate thread if it's not
+  # the sole purpose of the app.
+
+  # publish every minute
+  DispatchRider::ScheduledJob.publish_due_jobs every: 1.minute
+
+  # publish every half an hour
+  DispatchRider::ScheduledJob.publish_due_jobs every: 30.minutes
+  ```
+
```

Comment:
or add a lot more code to the AR adapter to do full change locking (I like using non locking, update based contention detection)

update table set x=1 where x=0; (this can be per row, so you know who won picking up that job for publishing)
{ask mysql how many rows were updated)
if one, you won, do your job, if 0, you lost, sleep for a bit and restart the attempt from scratch.